src/url.js

```diff
@@ -92,12 +92,15 @@ export function addParamsToUrl(url, params) {
  * Provides an exception for localhost.
  * @param {string} urlString
  * @param {!Element} elementContext Element where the url was found.
+ * @param {!Window=} opt_win
  * @return {string}
  */
-export function assertHttpsUrl(urlString, elementContext) {
+export function assertHttpsUrl(urlString, elementContext, opt_win) {
```

Comment:
Its only for tests. But your comment is correct. This was wrong in different way from what I thought.